browser/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "@0x/mesh-browser",
+    "version": "3.0.1-beta",
+    "description": "TypeScript and JavaScript bindings for running Mesh directly in the browser.",
+    "main": "./lib/index.js",
+    "license": "MIT",
+    "private": true,
+    "scripts": {
+        "build": "yarn build:go && yarn build:generate && yarn build:ts",
+        "build:ts": "node --max_old_space_size=4096 ./node_modules/.bin/tsc -b",
+        "build:generate": "go run ./scripts/generate_wasm_buffer.go",
+        "build:go": "GOOS=js GOARCH=wasm go build -o ./wasm/main.wasm ./go/main.go"
```

Comment:
Can we add `tslint`? See a bunch of things that the linter would have caught.

Reply:
👍just added `tslint`.